src/xhr.js

```diff
@@ -258,6 +288,10 @@ export function fetchPolyfill(input, opt_init) {
       reject(new Error('Request aborted'));
     };
 
+    if (init.responseType == 'document') {
```

Comment:
Actually, nevermind. Please just move it next to the `xhr.withCredentials = true;` above. `if (init.headers) {` should go there as well ideally.